src/toast/pybind11/pybind11/__init__.py

```diff
@@ -7,17 +7,17 @@ def get_include(user=False):
     import sys
 
     # Are we running in a virtual environment?
-    virtualenv = hasattr(sys, 'real_prefix') or \
```

Comment:
The pybind11, Random123, and googletest subdirectories are excluded from autoformatting by the format_source.sh script.  I'm guessing you just ran black recursively by itself?  I don't think we want to be autoformatting the source to external code that we have incorporated into our source tree.